indico/modules/auth/controllers.py

```diff
@@ -65,9 +66,13 @@ def _process(self):
             multipass.set_next_url()
             return multipass.redirect_success()
 
+        sso_redirect = True
+        if 'User-Agent' in request.headers:
+            sso_redirect = all(a not in request.headers['User-Agent'] for a in self.REDIRECT_EXEMPT)
```

Comment:
`not any(...)`

Reply:
I started with `not any` too 😄, but it boils down to the same right?

Reply:
Never mind, I've just re-read what I wrote shortly after my previous comment.